Python/sysmodule.c

```diff
@@ -1512,6 +1542,7 @@ static PyMethodDef sys_methods[] = {
     {"call_tracing", sys_call_tracing, METH_VARARGS, call_tracing_doc},
     {"_debugmallocstats", sys_debugmallocstats, METH_NOARGS,
      debugmallocstats_doc},
+    SYS_SET_COROUTINE_ORIGIN_TRACKING_DEPTH_METHODDEF
```

Comment:
Add `sys.get_coroutine_origin_tracking_depth`?

Reply:
Eh, I guess. I was thinking this is sufficiently arcane functionality that I could be lazy and get away with the `set`-returns-the-old-value pattern (cf. `signal.set_wakeup_fd`), but you're probably right that `set`+`get` is a nicer API.